### PR TITLE
[Twitter Adapter] Add unit tests for TwitterOptions class

### PR DESCRIPTION
**Pull request by sw-joelmut**
## Description
This PR adds unit tests for the [TwitterOptions](https://github.com/BotBuilderCommunity/botbuilder-community-dotnet/blob/develop/libraries/Bot.Builder.Community.Adapters.Twitter/TwitterOptions.cs#L9) class increasing its code coverage from **65%** to **100%**.

### Specific Changes
- Added the **TwitterOptionsTests** file with new unit tests.

## Testing
The following images show the new test passing and the resulting code coverage.
image

### CODE REVIEW

File: tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+﻿using System;
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs]: As this is a new file, I think we should include the Copyright header.

File: tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+using System.Collections.Generic;
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs]: Remove the unnecessary usings.

File: tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+using Moq;
+
+namespace Bot.Builder.Community.Adapters.Twitter.Tests
+{
+    [TestClass]
+    [TestCategory("Twitter")]
+    public class TwitterOptionsTests
+    {
+        [TestMethod]
+        public async Task TwitterOptionsProperties()
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs]: Why is this method async?

Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs]:
```suggestion
        public Task TwitterOptionsPropertiesShouldBeSetSuccessfully()
```

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

File: tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                AccessToken = "access-token",
+                AccessSecret = "access-secret",
+                Environment = "env",
+                WebhookUri = "uri",
+            };
+
+            Assert.AreEqual("bot-username", options.BotUsername);
+            Assert.AreEqual("usernames", options.AllowedUsernames[0]);
+            Assert.AreEqual(TwitterAccountApi.PremiumFree, options.Tier);
+            Assert.AreEqual(true, options.IsValid);
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs]:
```suggestion
            Assert.IsTrue(options.IsValid);
```

File: tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            Assert.AreEqual("consumer-key", options.ConsumerKey);
+            Assert.AreEqual("consumer-secret", options.ConsumerSecret);
+            Assert.AreEqual("access-token", options.AccessToken);
+            Assert.AreEqual("access-secret", options.AccessSecret);
+            Assert.AreEqual("env", options.Environment);
+            Assert.AreEqual("uri", options.WebhookUri);
+            Assert.AreEqual(true, options.AllowedUsernamesConfigured());
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/TwitterOptionsTests.cs]:
```suggestion
            Assert.IsTrue(options.AllowedUsernamesConfigured());
```

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

**Review by ceciliaavila — APPROVED** [review]

**Review by santgr11 — APPROVED** [review]

**Review by ian-whiteout — APPROVED** [review]

**Pull request closed without merging**